src/cloud-element-templates/Linter.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import StaticResolver from 'bpmnlint/lib/resolver/static-resolver';
+import ElementTemplates from './ElementTemplates';
+import { getPropertyValue, validateProperty } from './util/propertyUtil';
+
+import { applyConditions } from './Condition';
+
+import BpmnModdle from 'bpmn-moddle';
+import zeebeModdle from 'zeebe-bpmn-moddle/resources/zeebe';
+
+import { Validator } from './Validator';
+
+export const elementTemplateLintRule = ({ templates = [] }) => {
+  const moddle = new BpmnModdle({ zeebe: zeebeModdle });
+
+  const validator = new Validator(moddle).addAll(templates);
+  const validTemplates = validator.getValidTemplates();
+
+  // We use the ElementTemplates Module without the required bpmn-js modules
+  // As we only use it to facilitate template ID and version lookup,
+  // access to commandstack etc. is not required
+  const elementTemplates = new ElementTemplates();
+  elementTemplates.set(validTemplates);
+
+  function check(node, reporter) {
+
+    let template = elementTemplates.get(node);
+
+    const templateId = elementTemplates._getTemplateId(node);
+
+    // Handle missing template
+    if (templateId && !template) {
+      reporter.report(
+        node.id,
+        'Linked element template not found',
+        {
+          name: node.name
+        }
+      );
+      return;
+    }
+
+    if (!template) {
+      return;
+    }
+
+    template = applyConditions(node, template);
+
+    // Check attributes
+    template.properties.forEach((property) => {
+      const value = getPropertyValue(node, property);
+      const error = validateProperty(value, property);
+
+      if (!error) {
+        return;
+      }
+
+      reporter.report(
+        node.id,
+        `${property.label} ${firstLetterToLowerCase(error)}`,
+        {
+          entryIds: [ getEntryId(property, template) ],
```

Comment:
The structure of a report should be

```json
{
  "propertiesPanel": {
    "entryId": "foo"
  }
}
```

according to [@camunda/linting](https://github.com/camunda/linting/blob/main/lib/Linter.js#L77). I noticed that this structure is also assumed [here](https://github.com/camunda/linting/commit/278d4a6417920694c158411ee70b04c714b20ef8). I'd like to change this so we have one structure instead of two.

Reply:
I initially decided against this as one report can [refer to multiple entries ](https://github.com/camunda/linting/blob/main/lib/utils/properties-panel.js#L22-L43)in the properties panel (eg. [duplicated property error](https://github.com/camunda/linting/blob/main/lib/utils/properties-panel.js#L163-L167)), which is why entryIds is an array.
The entryId set in the report which is used to focus is just the first from the array. Just adding a single entryId would not allow linter plugins to reference multiple property panel entries. 

We could nest the `entryIds` option into the propertiesPanel and keep it side-by-side with the `entryId`:
```
propertiesPanel: {
  entryIds: ["foo", "bar"]
  entryId: "foo"
}
```
Alternatively, we can always keep the full list of `entryIds` for the properties panel and select the first one when we fire the event [here](https://github.com/camunda/linting/blob/bba0f7419c56e8b2024934ab60c4ec00164df4e4/lib/modeler/Linting.js#L39-L41). What do you think?

Reply:
For 2️⃣ , I created a draft PR in linting with the proposed changes: https://github.com/camunda/linting/pull/63

Reply:
I adjusted the PR to aligned with 2️⃣  as of [201c4ec](https://github.com/bpmn-io/bpmn-js-element-templates/pull/3/commits/201c4ecbde9c8575f9bf0dd86704b1b6d9b84892). Together with https://github.com/camunda/linting/pull/63, this will align our report structure and allows linter plugins to supply more than one entryId